app/routes.py

```diff
@@ -14,6 +14,7 @@
 from app.modules.historicalmodels import *
 from app.modules.constraintmodel import *
 from app.modules.regionmodel import *
+from app.modules.sensorModel import *
```

Comment:
Can you instead make this `import app.modules.sensorModel`? You only need to use `from` when there is a specific class you want from the package